packages/fiori-freestyle-writer/test/worklist.test.ts

```diff
@@ -121,6 +123,25 @@ describe(`Fiori freestyle template: ${TEST_NAME}`, () => {
                     metadata: getMetadata('fe_samples_v4')
                 }
             }
+        },
+        {
+            name: 'worklist_ts',
+            config: {
+                ...commonConfig,
+                template: {
+                    type: TemplateType.Worklist,
+                    settings: {
+                        entity: {
+                            name: 'SEPMRA_C_PD_Product',
+                            key: 'Product',
+                            idProperty: 'Name',
+                            numberProperty: 'Price',
+                            unitOfMeasureProperty: 'Currency'
+                        }
+                    } as WorklistSettings
+                },
+                service: v2Service
```

Comment:
This test is not using typescript. 
```
appOptions:{
                    typescript: true
                }
```
Adding above. Navigation to object page fails as there bug in packages/fiori-freestyle-writer/test/test-output/worklistTemplate/worklist_ts/webapp/controller/Object.controller.ts 

````

        var oResourceBundle = this.getResourceBundle(),
            oObject = oView.getBindingContext().getObject(),
            sObjectId = oObject.,
            sObjectName = oObject.;
````
VS Code shows various problems for the file
<img width="1266" alt="Screenshot 2022-08-25 at 17 07 35" src="https://user-images.githubusercontent.com/10102054/186715321-5371d354-7cc0-4953-adcf-123c11ff671b.png">

Reply:
That is odd. I played with the output of all tests when using the `UX_DEBUG` variable. I will have a look at it again.
Thanks for spotting it!

Reply:
Test fixed with https://github.com/SAP/open-ux-tools/pull/626/commits/a0bf107ec87c049fcdee46eba2bd140967a918ea and object page issue fixed with https://github.com/SAP/open-ux-tools/pull/626/commits/9c5d5d3ad74d54c8d95c528391ff91271dc21de0

Reply:
> VS Code shows various problems for the file

Improved with https://github.com/SAP/open-ux-tools/pull/626/commits/ad1a11535ce97c041714ad8ec7e7cb5f5d790909.

Reply:
Thanks